src/components/propertiesInput/PropertiesInputData.jsx

```diff
@@ -1,170 +1,251 @@
-                  <select
-                    name="purpose"
-                    id="choices"
-                    className="property-option">
-                    <option value="" disabled selected hidden>
-                      Choose property option
-                    </option>
-                    <option value="rent">FOR RENT</option>
-                    <option value="sale">FOR SALE</option>
-                  </select>
-                </div>
-                <div className="property-data-input-field">
-                  <select
-                    name="purpose"
-                    id="choices"
-                    className="property-option">
-                    <option value="" disabled selected hidden>
-                      Bedrooms
-                    </option>
-                    {PropertyFeature.map((feature) => (
-                      <option value="feature">{feature}</option>
-                    ))}
-                  </select>
-                </div>
-                <div className="property-data-input-field">
-                  <select
-                    name="purpose"
-                    id="choices"
-                    className="property-option">
-                    <option value="" disabled selected hidden>
-                      Bathroom
-                    </option>
-                    {PropertyFeature.map((feature) => (
-                      <option value="feature">{feature}</option>
-                    ))}
-                  </select>
-                </div>
-                <div className="property-data-input-field">
-                  <select
-                    name="purpose"
-                    id="choices"
-                    className="property-option">
-                    <option value="" disabled selected hidden>
-                      Garage
-                    </option>
-                    {PropertyFeature.map((feature) => (
-                      <option value="feature">{feature}</option>
-                    ))}
-                  </select>
-                </div>
+            <div className="properties-content-wrapper">
+              <label htmlFor="title">Title</label>
+              <InputField
+                type='text'
+                placeholder='Title'
+                name='title'
+                value={propertiesInfo.title}
+                onChange={informationHandler}
+              />
+              <label htmlFor="yearBuild">Title</label>
+              <InputField
+                type='text'
+                placeholder='yearBuild'
+                name='yearBuild'
+                value={propertiesInfo.yearBuild}
+                onChange={informationHandler}
+              />
+              <label htmlFor="price">Title</label>
+              <InputField
+                type='text'
+                placeholder='price'
+                name='price'
+                value={propertiesInfo.price}
+                onChange={informationHandler}
+              />
```

Comment:
I suggest you map the Input so you can reduce the lines of code here and also stick to DRY principle